containers/ecr-viewer/src/app/admin/user/UserForm.tsx

```diff
@@ -0,0 +1,288 @@
+          onChange={(e) => setEmail(e.target.value)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const UserTypeFieldSet = ({
+  userType,
+  setUserType,
+}: {
+  userType: UserType;
+  setUserType: (n: UserType) => void;
+}) => {
+  return (
+    <FieldSet legend="User type">
+      <label className="display-block width-full">
+        USER TYPE
+        <RequiredMarker />
+        {[
+          {
+            name: "admin",
+            description:
+              "Admins have full access to user management, program management, and the eCR Library",
+          },
+          {
+            name: "standard",
+            description:
+              "Standard users can only use the eCR Library with limited access to program area(s)",
+          },
+        ].map((option) => (
+          <Radio
+            key={option.name}
+            label={toTitleCase(option.name)}
+            labelDescription={option.description}
+            type="radio"
+            required={true}
+            name="userType"
+            id={`userType-${option.name}`}
+            value={option.name}
+            checked={userType === option.name}
+            onChange={(e) => setUserType(e.target.value as UserType)}
+          />
+        ))}
+      </label>
+    </FieldSet>
+  );
+};
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export const ProgramFieldSet = ({
+  programs,
+  setPrograms,
+  numProgramsSelected,
+  userType,
+}: {
+  programs: FormProgram[];
+  setPrograms: (c: FormProgram[]) => void;
+  numProgramsSelected: number;
+  userType: UserType;
+}) => {
+  const [expandedPrograms, setExpandedPrograms] = useState<
+    Record<string, boolean>
+  >(valsToBoolean(programs, true));
+  const isStandardUser = userType === "standard";
+
+  const accordionItems: AccordionItem[] = programs.map((program) => {
+    const { name, conditions } = program;
+    const numConditions = conditions.length;
+
+    return {
+      title: (
+        <div className="display-flex flex-justify flex-align-center">
+          <span>{name}</span>
+          <span>
+            {numConditions} condition{makePlural(numConditions)}
+          </span>
+        </div>
+      ),
+      content: conditions.map((c) => c.condition_name).join("; "),
+      id: toKebabCase(name),
+      expanded: !!expandedPrograms[toKebabCase(name)],
+      headingLevel: "h3",
+      group: "program",
+      isChecked: () => isStandardUser && program.checked === true,
+      onChecked: isStandardUser
+        ? (e: React.ChangeEvent<HTMLInputElement>) => {
+            setPrograms(
+              programs.map((c) =>
+                c.uuid === program.uuid
+                  ? { ...c, checked: e.target.checked }
+                  : c,
+              ),
+            );
+          }
+        : undefined,
+    };
+  });
+
+  return (
+    <FieldSet legend="Program area access">
+      <span>
+        Select one or more program areas. Program areas will apply to all users
```

Comment:
@angelathe this text also needs to be updated if the user is an admin as we aren't asking the creator to select anything, but instead should say something about admins having access to all program areas